survey-sdk/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest package="co.nimblehq.lib">
```

Comment:
I think we need a better package name for this lib, for example: `co.nimblehq.survey.sdk` 🤔

Reply:
@luongvo good idea, I forgot this. It will be updated in PR #9